lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -264,17 +264,17 @@ def self.append_phc_version_if_necessary(append_phc_version_if_next_version_is_n
 
       # rubocop:disable Metrics/PerceivedComplexity
       private_class_method def self.get_section_depending_on_types_of_change(change_types)
-        if change_types.include?("breaking")
+        if change_types.any?("pr:breaking")
```

Comment:
Hmm why the change from `include?` to `any?`

Reply:
oopsies, this is from before, where it could be with or without prefix. Good catch